src/mds/MDSAuthCaps.cc

```diff
@@ -349,12 +349,13 @@ bool MDSAuthCaps::parse(CephContext *c, std::string_view str, ostream *err)
     return true;
   }
 
+  if (c != nullptr)
```

Comment:
Why we started setting the member conditionally? This excludes the ability to nullify `cct` (abstracting whether this feature is needed or not).

Reply:
`cct` is initialized to `nullptr` in `MDSAuthCaps.h`. So I was skipping assignment. But, yeah, it's unnecessary.